helper/helper.go

```diff
@@ -245,7 +258,7 @@ func getOidListFromFile(oidFileName string) ([]int, error) {
 	return oidList, nil
 }
 
-func flushAndCloseRestoreWriter(pipeName string, oid int) error {
+func (h *Helper) flushAndCloseRestoreWriter(pipeName string, oid int) error {
```

Comment:
This function can also be moved to `helper/restore_helper.go`, because `writer` and `writeHandle` are used only there.

Reply:
in backup helper `writer` and `writeHandle` are always `nil`